Added detailed instructions to README.rst

astrofrog opened the pull request:
This cleans up and adds the full instructions to the README file. I originally thought I'd add it to the astropy dev docs but that actually doesn't seem right because we should just keep the instructions close to the benchmarks.

[Preview here](https://github.com/astropy/astropy-benchmarks/blob/758dcab794907951f399009169a65f07cff54240/README.rst)

bsipocz commented on README.rst:
```diff
@@ -41,21 +89,58 @@ and running the following in the astropy-benchmarks folder::
-You do not need to add any result files for the benchmarks - we have machines that automatically do this every night.
+Once you are happy with your benchmark(s), open a pull request to the
+astropy-benchmarks repository. You do not need to add any result files for the
+benchmarks - we have machines that automatically do this every night.
```
No need for the google docs any more, but I cannot comment on those lines below

bsipocz commented on README.rst:
```diff
@@ -4,25 +4,73 @@ Astropy performance benchmarks
+
+then clone the benchmarks repository::
+
+    git clone git@github.com:astropy/astropy-benchmarks.git
```
you may want to mention the `--single-branch` cloning option in case someone only wants to run some benchmarks locally without reporting the results, etc.

astrofrog replied:
Done

bsipocz commented on README.rst:
```diff
@@ -41,21 +89,58 @@ and running the following in the astropy-benchmarks folder::
-You do not need to add any result files for the benchmarks - we have machines that automatically do this every night.
+Once you are happy with your benchmark(s), open a pull request to the
+astropy-benchmarks repository. You do not need to add any result files for the
+benchmarks - we have machines that automatically do this every night.
```
Please add a short note about how to view the results, even if it's just pointing to the relevant asv docs pages ( https://asv.readthedocs.io/en/stable/using.html#viewing-the-results)

astrofrog replied:
Done

bsipocz commented:
Very much agree that these are better placed here than in the astropy docs. I've left a few minor comments.

astrofrog commented:
@bsipocz - let me know if the latest changes address your comments

bsipocz commented:
@astrofrog - You can now remove the link to the google docs that I added yesterday. Otherwise it looks great to me, thanks.

astrofrog commented:
@pllim - I think I've addressed all your comments. I've not mentioned the timeout thing since that's not actually really needed (it just depends how you set up the cron job)
